src/include/btmem.h

```diff
@@ -214,9 +218,9 @@ struct __wt_ovfl_reuse {
 #endif
 #define WT_HS_CONFIG                                                              \
     "key_format=" WT_UNCHECKED_STRING(IuQQ) ",value_format=" WT_UNCHECKED_STRING( \
-      QQBu) ",block_compressor=" WT_HS_COMPRESSOR                                 \
-            ",leaf_value_max=64MB"                                                \
-            ",prefix_compression=false"
+      QQBuBQ) ",block_compressor=" WT_HS_COMPRESSOR                               \
```

Comment:
I made a suggestion in the JIRA ticket - I think we should still make a change here - but just changing the `upd_type` to be a `uint64_t`.

Reply:
2404265